enterprise/web.core/src/org/netbeans/modules/web/core/palette/items/IfCustomizer.java

```diff
@@ -159,6 +159,11 @@ private void initComponents() {
         jLabel2.getAccessibleContext().setAccessibleName(org.openide.util.NbBundle.getMessage(IfCustomizer.class, "ACSN_If_Variable")); // NOI18N
         jLabel2.getAccessibleContext().setAccessibleDescription(org.openide.util.NbBundle.getMessage(IfCustomizer.class, "ACSD_If_Variable")); // NOI18N
 
+        jComboBox2.addActionListener(new java.awt.event.ActionListener() {
+            public void actionPerformed(java.awt.event.ActionEvent evt) {
+                jComboBox2ActionPerformed(evt);
+            }
+        });
```

Comment:
Why?

Reply:
Don't know exactly what happend. Going to remove that.